components/Book/index.js

```diff
@@ -1,27 +1,11 @@
 import Image from "next/image";
 
 export default function Book({ book }) {
-  const {
-    imageURL,
-    ISBN,
-    EAN,
-    language,
-    pagenumber,
-    publication,
-    price,
-    description,
-  } = book;
+  const { imageURL, title } = book;
 
   return (
     <div>
       <Image src={imageURL} alt={title} width={150} height={200} />
```

Comment:
All your book images are distorted because each jpeg has another dimension and ratio but in the code you are using a fixed dimension for all of them.

Reply:
What do you recommend me for this? @josch87

Reply:
There are three options as long as you don't want the user to upload images:
- Cropping out each image with photoshop and bringing it all in the same dimensions
- Finding a library that cuts the images (but then it is not in your control what exactly is beeing displayed and some images might be unfavorably cropped)
- Save the dimensions of every image to your data and use these for the width and height props of the Image-component
  - Maybe you find a way to automate this when you have a lot of images

Reply:
Maybe I have the solution, the idea of which would be:
to create a frame and wrap each book inside the frame...Let me spend some time with this 😄